#1. Plantear la hipotesis

Definimos las hipótesis para este estudio:

Hipótesis Nula ($H_0$): No hay diferencia en la reducción de la presión arterial sistólica (PAS) entre el grupo de tratamiento (medicamento) y el grupo de control (placebo). Matemáticamente, esto lo expresaremos como:

$H_0:μ_{medicamento}=μ_{placebo}$

Hipótesis Alternativa ($H_1$): Hay una diferencia en la reducción de la PAS entre el grupo de tratamiento y el grupo de control. Esto lo expresaremos como:

$H_1:μ_{medicamento}≠μ_{placebo}$

Dado que queremos saber si el medicamento tiene algún efecto, sin especificar si aumentará o disminuirá la PAS en comparación con el placebo, este es un caso de prueba de dos colas.



#2. Seleccionar la prueba estadística apropiada

Debido a que queremos comparar las medias de dos muestras independientes (grupo de tratamiento y grupo de control) y no conocemos las desviaciones estándar de la población, una prueba t para muestras independientes es adecuada para este caso. La función ttest_ind de scipy.stats es ideal para realizar este análisis.

#3. Calcular el estadístico de prueba y el valor p

In [ ]:
import numpy as np
from scipy.stats import ttest_ind

# Datos de los dos grupos dados
grupo_medicamento = np.array([130, 125, 140, 118, 124, 134, 128, 136, 130, 129])
grupo_placebo = np.array([140, 142, 138, 150, 139, 142, 137, 148, 142, 141])

statistic, p_value = ttest_ind(grupo_medicamento, grupo_placebo)
print("Estadístico t:", statistic)
print("Valor p:", p_value)


Estadístico t: -5.2341436259783665
Valor p: 5.6088053781073984e-05


# 4. Interpretar los resultados: Compara el valor p con un nivel de significancia (α) predefinido (ej. α = 0.05). ¿Rechazas o no rechazas la hipótesis nula? ¿Qué conclusiones puedes sacar sobre la eficacia del nuevo medicamento?

Dado el valor de estadístico t de -5.23 y un valor p de aproximadamente 5.6088053781073984e-05, podemos interpretar los resultados de la siguiente manera:
Comparación del valor p con el nivel de significancia:
Si consideramos un nivel de significancia
𝛼=0.05, el valor p obtenido (5.6088053781073984e-05) es mucho menor que 0.05.
Entonces como conclusión podemos decir que rechazamos la hipótesis nula ($H_0$), ya que el valor p es significativamente menor que nuestro nivel de significancia de 0.05. Esto sugiere que existe una diferencia estadísticamente significativa en la reducción de la presión arterial sistólica (PAS) entre el grupo de tratamiento (que recibe el medicamento) y el grupo de control (que recibe el placebo). Por lo tanto, concluimos que el nuevo medicamento es eficaz en la reducción de la PAS en comparación con el placebo. Estos resultados apoyan la hipótesis de que el medicamento tiene un efecto significativo en la disminución de la presión arterial en pacientes hipertensos.
Este análisis sugiere que el medicamento podría ser útil como tratamiento para reducir la presión arterial en personas con hipertensión.